docs/operation/changing_control_plane_ingress_ip.rst

```diff
@@ -0,0 +1,44 @@
+Changing the Control Plane Ingress IP
+=====================================
+
+#. On the bootstrap node, update the ``ingressIP`` field from
+   ``networks.controlPlane`` in the Bootstrap configuration file.
+   (refer to :ref:`Bootstrap Configuration<Bootstrap Configuration>`)
+
+#. Refresh the pillar.
+
+   .. code-block:: console
+
+     $ salt-call saltutil.refresh_pillar wait=True
+
+#. Check that the change is taken into account.
+
+   .. code-block:: console
+
+      $ salt-call metalk8s_network.get_control_plane_ingress_ip
+      local:
+          <my-new-ip>
+
+#. On the bootstrap node, reconfigure ApiServer:
+
+   .. parsed-literal::
+
+     $ salt-call state.sls \\
+         metalk8s.kubernetes.apiserver \\
+         saltenv=metalk8s-|version|
+
+#. Reconfigure Control Plane conponents:
+
+   .. parsed-literal::
+
+      $ kubectl exec -n kube-system -c salt-master \\
+          --kubeconfig=/etc/kubernetes/admin.conf \\
+          $(kubectl --kubeconfig=/etc/kubernetes/admin.conf get pod \\
+          -l "app.kubernetes.io/name=salt-master" \\
+          --namespace=kube-system -o jsonpath='{.items[0].metadata.name}')  \\
```

Comment:
`-o name`?

Reply:
Hmm right I could use it :thinking: 
It will have strange behavior if for whatever reason you have multiple salt-master running, maybe better to keep this one for the moment (so that this procedure may be compatible with future Bootstrap HA :smile: :wink: )

Reply:
Haha good point :ok_hand: We'll have to make these commands simpler anyway, so if we change how we deploy things, we don't have to update it in hundreds of places across our docs.